language/functional_tests/tests/testsuite/builtins/create_account_and_get_sequence_number.mvir

```diff
@@ -1,22 +0,0 @@
-import 0x0.LibraAccount;
-import 0x0.LibraCoin;
-
-main() {
-    let addr: address;
-    let account_exists: bool;
-    let ten_coins: LibraCoin.T;
-    let account2_sequence_number: u64;
-
-    addr = 0x0111111111111111111111111111111111111011111111111111111111111110;
-    account_exists = LibraAccount.exists(copy(addr));
-    assert(!move(account_exists), 83);
-
-    ten_coins = LibraAccount.withdraw_from_sender(10);
-    create_account(copy(addr));
-    LibraAccount.deposit(copy(addr), move(ten_coins));
-
-    account2_sequence_number = LibraAccount.sequence_number(move(addr));
-    assert(copy(account2_sequence_number) == 0, 84);
```

Comment:
We could probably just add this assertion to the other `create_account` test?

Reply:
added in the previous one as suggested